backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Delete, Get, Param } from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('user')
```

Comment:
ogólna konwencja przewiduje raczej używanie liczby mnogiej w przypadku opisywania zasobów (czyli users zamiast user) chyba że w aplikacji byłby tylko jakiś 1 konkretny user to wtedy liczba pojedyncza by była adekwatna